test/jdk/javax/sql/resourceBundleTests/ValidateGetBundle.java

```diff
@@ -0,0 +1,82 @@
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import com.sun.rowset.JdbcRowSetResourceBundle;
+
+import java.util.Locale;
+import java.util.MissingResourceException;
+import java.util.ResourceBundle;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.Arguments;
+
+/**
+ * @test
+ * @bug 8294989
+ * @summary Check JDBC RowSet resource bundle access
+ * @modules java.sql.rowset/com.sun.rowset:+open
+ * @run junit/othervm ValidateGetBundle
+ */
+public class ValidateGetBundle{
+
+    // Resource bundle base name via a fully qualified class name
+    private static final String FULLY_QUALIFIED_CLASS_NAME = "com.sun.rowset.RowSetResourceBundle";
+    // Resource bundle base name via a path
+    private static final String PATH_TO_BUNDLE = "com/sun/rowset/RowSetResourceBundle";
+
+    /**
+     * Test to validate whether the JDBC RowSet Resource bundle can be found
+     * @param bundleName the base name of the resource bundle
+     * @param expectBundle indicates whether the resource bundle should be found
+     */
+    @ParameterizedTest
+    @MethodSource("bundleProvider")
+    void testResourceBundleAccess(String bundleName, boolean expectBundle) {
+        try {
+            var bundle = ResourceBundle.getBundle(bundleName,
+                    Locale.US, JdbcRowSetResourceBundle.class.getModule());
+            if (!expectBundle) {
+                throw new RuntimeException(
+                        String.format("$$$ Error: '%s' shouldn't have loaded!%n", bundleName));
+            }
+            System.out.printf("$$$ %s was found as expected!%n", bundleName);
+        } catch (MissingResourceException mr) {
+            if (expectBundle) {
+                throw new RuntimeException(
+                        String.format("$$$ Error: '%s' should have loaded!", bundleName), mr);
+            }
+            System.out.printf("$$$ %s was not found as expected!", bundleName);
+        }
+    }
+
+    // Data provider for testResourceBundleAccess
+    private static Stream<Arguments> bundleProvider() {
+        return Stream.of(
+                // The resource bundle should be found with the fully qualified class name
+                Arguments.of(FULLY_QUALIFIED_CLASS_NAME, true),
+                // The resource bundle will not be found when the path is specified
+                Arguments.of(PATH_TO_BUNDLE, false)
+        );
+    }
```

Comment:
I think the literals can be used in `bundleProvider` - no need to define constants.
Also, `@DataProvider`s are typically located towards the beginning of the file.

Reply:
I am fine with the constants as I find it easier to see/modify/document than within the DataProvider.

This is really a style choice so your milage may vary :-)